### PR TITLE
Add emanote module

### PR DESCRIPTION
**Pull request by srid**
[I recommend squash merging; unclean commit history!]

<details>
Getting a strange error:

```
❯ nix build
warning: input 'mission-control' has an override for a non-existent input 'nixpkgs'
error: Couldn't figure out flakeref for emanote: https://github.com/srid/emanote
(use '--show-trace' to show detailed location information)
```
</details>

- [x] Fix option docs

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -32,6 +34,17 @@
           '';
         };
 
+        emanote = {
+          baseUrl = "https://github.com/srid/emanote";
```
Comment (roberth) [inline flake.nix:38]:
```suggestion
          baseUrl = "https://github.com/srid/emanote/blob/master";
```

Some code parses this. It should take the same information from specific individual options instead. I'll improve that later. For now this should make it work.

**Comment by roberth** [comment]
It relies on modules having a file name that's in the flake directory, but this info isn't available because `import` hides it at https://github.com/srid/emanote/blob/6ae624c026f586984acadbc977b412b8577990a5/flake.nix#L119

If you remove `import`, the docs should get rendered.

**Comment by srid** [comment]
Hmm, yes. The fact that with `import` it was always working before (in https://github.com/srid/emanote-template) is ... strange.

Okay, fixed .. but now I see:

```
❯ nix build
warning: input 'mission-control' has an override for a non-existent input 'nixpkgs'
error: The option `systems' is used but not defined.
(use '--show-trace' to show detailed location information)
```

The stack trace contains `std` module for some reason. If I remove `std` entirely:

```
❯ nix build
warning: Git tree '/Users/srid/code/flake.parts-website' is dirty
warning: input 'mission-control' has an override for a non-existent input 'nixpkgs'
error: attribute 'inputs' missing

       at /nix/store/8zrjcl2ssb4b83m7wpqyvmj004h7bx6p-source/modules/perSystem.nix:95:75:

           94|         config = {
           95|           _module.args.inputs' = mapAttrs (k: rootConfig.perInput system) self.inputs;
             |                                                                           ^
           96|           _module.args.self' = rootConfig.perInput system self;
(use '--show-trace' to show detailed location information)
```

**Comment by roberth** [comment]
It seems that `std` has some logic that's always enabled, which makes the trace longer, but the root cause is a missing `defaultText` in the emanote `package` option.

**Pull request merged**